components/_global-navigation.scss

```diff
@@ -193,12 +210,24 @@ $notification-counter-size: 18px;
 		flex: 0 1 auto;
 	}
 
+	&__search {
+		flex: 0 1 auto;
+		transition: flex $search-transition-duration;
+	}
+
 	&__search-input-wrapper {
-		border: 0 solid $color-slate-gray;
+		border: {
+			bottom: 1px solid transparent;
+			left: 0;
+			right: 0;
+			top: 0;
+		};
 		display: flex;
 		// Safari doesn't set default flex values if we use `flex: 1;`
 		// it requires us to put these values explicitly.
 		flex: 1 1 auto;
+		min-height: 28px;
```

Comment:
We could set just `height`, but it's needed to keep the same size of the wrapper whether it's closed, open with or without buttons. But I'll give it another check.